seat/modules/power_module.py

```diff
@@ -320,8 +320,23 @@ def reset_bc_data_order(bc_data):
         bc_data = bc_data.sort()
 
 
-def roundup(x, base=5):
-    return base * round(x/base)
+def roundup_placevalue(x, decimal=5):
+    #Round up to the nearest place
+    # +decimal = round up to the defined decimal place
+    # -decimal = round up to the nearest 10
+    # e.g. for value 100.12345, decimal -3 = 1000, decimal 3 = 100.124
+    decimal = float(decimal)
+    if decimal >= 0:
+        rounded = np.ceil(x * 10**decimal) / 10**decimal
+    else:
+        rounded = np.ceil(x / 10**(-1*decimal)) * 10**(-1*decimal)
+    return rounded
+
+def roundup(x, val=2):
+    rounded = np.ceil(x)
+    while np.mod(rounded, val) != 0:
+        rounded += 1
+    return rounded
```

Comment:
What about the following which eliminates the need for a while loop?
```
def roundup(x, val=2):
    return np.ceil(x / val) * val

# Test
print(roundup(7, 2))  # Output: 8.0
print(roundup(7, 5))  # Output: 10.0
```

Reply:
@ssolson Your code works great, feel free to switch it to that. roundup_placevalue def can be removed. I didn't remove it after testing which roundup should have been used for plotting. Let me know if you want me to make the changes or if you plan to. 

Reply:
Tim could you review and approve 
https://github.com/IntegralEnvision/seat-qgis-plugin/pull/6
when you get a chance?

Reply:
@ssolson Looks good. I merged your commits.